llvm/test/DebugInfo/X86/debug-aranges-sce-tuning.ll

```diff
@@ -0,0 +1,33 @@
+; This checks that .debug_aranges is always generated for the SCE debugger
```

Comment:
The https://llvm.org/docs/TestingGuide.html#elaborated-tests style could make test update easier in the future.

Reply:
Done!